nft_ingester/src/api/error.rs

```diff
@@ -38,8 +38,8 @@ pub enum DasApiError {
     Validation(String),
     #[error("Page number is too big. Up to {0} pages are supported with this kind of pagination. Please use a different pagination(before/after/cursor).")]
     PageTooBig(usize),
-    #[error("Internal DB error")]
-    InternalDbError,
+    #[error("Internal Postgres error")]
+    InternalPostgresError,
```

Comment:
Are we sure no-one is relying on the text in these errors?

Reply:
yep, this error was added by @andrii-kl with the first iteration of healthchecks

Reply:
I suggested create a more specific error if we know which database is the problem. 
This will help quickly understand what needs attention.
I think we shouldn't remove **InternalDbError** just add more specific  errors
**RocksDbError** and **PgDbError**

Reply:
there are no usages for it outside of healthchecks.